ports/stm32/sdram.c

```diff
@@ -276,6 +276,14 @@ void sdram_leave_low_power(void) {
         (0 << 9U));                                                  // Mode Register Definition
 }
 
+void sdram_enter_power_down(void) {
```

Comment:
Should there be (can there be?) an `sdram_leave_power_down()` function as well, in case a board wants to use this during stop mode?

Also, unless I'm reading it wrong, this function looks exactly the same as `sdram_enter_low_power()`.

Reply:
> Should there be (can there be?) an sdram_leave_power_down() function as well, in case a board wants to use this during stop mode?
When the device is selected it exits the power-down mode, and goes into normal mode, so technically it's not needed, but `sdram_leave_low_power()` can be used.

> Also, unless I'm reading it wrong

Sorry copy&paste typo, fixed.